bin/phpbench.bat

```diff
@@ -0,0 +1,2 @@
+@echo off
+php.exe "%~dp0\phpbench.php" %*
```

Comment:
Is this file necessary? I notice that well established libraries such as phpspec do not have this file. https://github.com/phpspec/phpspec/tree/master/bin